.github/workflows/test-install.yml

```diff
@@ -0,0 +1,153 @@
+name: test-install
+
+on:
+  push:
+    branch:
+      - master
+      - develop
+      - release/*
+      - dm/*
+    paths:
+      - 'setup.*'
+      - 'environment.yml'
+      - 'requirements*.txt'
```

Comment:
this will get stuff in `requirements/` correct?
one could make this more explicit

Reply:
No, it actually matches for all files that have a name that matches this pattern across the whole repository.

Reply:
Ok, fine